FEXCore/Source/Interface/IR/IR.h

```diff
@@ -607,14 +610,36 @@ static inline uint8_t OpSizeToSize(IR::OpSize Size) {
   }
 }
 
+static inline uint16_t OpSizeAsBits(IR::OpSize Size) {
+  LOGMAN_THROW_A_FMT(Size != IR::OpSize::iInvalid, "Invalid Size");
+  return IR::OpSizeToSize(Size) * 8u;
+}
+
 static inline OpSize MultiplyOpSize(IR::OpSize Size, uint8_t Multiplier) {
+  LOGMAN_THROW_A_FMT(Size != IR::OpSize::iInvalid, "Invalid Size");
   return IR::SizeToOpSize(IR::OpSizeToSize(Size) * Multiplier);
 }
 
 static inline OpSize DivideOpSize(IR::OpSize Size, uint8_t Multiplier) {
```

Comment:
`Multiplier` should probably be `Divisor` here. Though this function seems like it can be merged into the operator variant down below

Reply:
Ah right, renamed the variable.
Sadly can't yet be folded in to the division operator overload yet since that causes ambiguous compile errors until we can convert the enum to an enum class.